ushahidi/platform-client
# Improve categories

**Dependent of https://github.com/ushahidi/platform/pull/1670 to work **
This pull request makes the following changes:
- Adding new ui for adding and updating categories
- Adding possibility to add categories as a field in surveys
- Updating ui for categories when displaying posts
- Displaying category-filter in mode-context-form-filter

Test these changes by:
- Add new category
- Add a new survey, choose to add a new category-field or task with a category-field 
- Try adding to the survey
- Look at the post, the categories should display
- Look at left-hand-filter, categories should display

Fixes ushahidi/platform#1397 .

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/616)
